include/DataFile.h

```diff
@@ -122,14 +123,17 @@ class EXPORT DataFile : public QDomDocument
 	void upgrade_0_4_0_20080622();
 	void upgrade_0_4_0_beta1();
 	void upgrade_0_4_0_rc2();
+	void upgrade_1_0_99();
 	void upgrade_1_1_0();
 	void upgrade_1_1_91();
 
 	void upgrade();
 
 	void loadData( const QByteArray & _data, const QString & _sourceFile );
 
+	void findIds(const QDomElement& elem, QList<jo_id_t>& idList);
```

Comment:
This function does not depend on the class. It should be a static function in `DataFile.cpp`.